### PR TITLE
chore: reflect actual policy about Committers' rights

### PR DESCRIPTION
**Pull request by KisaragiEffective**
## What

権限の運用実態に合わせてissueの管理方法を追記

## Why

内部的な議論による

## Checklist
- [x] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [ ] Test working in a local environment
- [ ] (If needed) Add story of storybook
- [ ] (If needed) Update CHANGELOG.md
- [ ] (If possible) Add tests

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,7 +53,7 @@ Thank you for your PR! Before creating a PR, please check the following:
 - If possible, prefix the title with a keyword that identifies the type of this PR, as shown below.
   - `fix` / `refactor` / `feat` / `enhance` / `perf` / `chore` etc
   - Also, make sure that the granularity of this PR is appropriate. Please do not include more than one type of change or interest in a single PR.
-- If there is an Issue which will be resolved by this PR, please include a reference to the Issue in the text.
+- If there is an Issue which will be resolved by this PR, please tie issues in the text.
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: This was fine before now it just has broken english

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
...
+
 When you want to add a feature or fix a bug, **first have the design and policy reviewed in an Issue** (if it is not there, please make one). Without this step, there is a high possibility that the PR will not be merged even if it is implemented.
 
 At this point, you also need to clarify the goals of the PR you will create, and make sure that the other members of the team are aware of them.
 PRs that do not have a clear set of do's and don'ts tend to be bloated and difficult to review.
 
-Also, when you start implementation, assign yourself to the Issue (if you cannot do it yourself, ask another member to assign you). By expressing your intention to work the Issue, you can prevent conflicts in the work.
+Also, when you start implementation, assign yourself to the Issue (if you cannot do it yourself, ask Commiter to assign you). By expressing your intention to work the Issue, you can prevent conflicts in the work.
+
+To the Commiters: you should not assign someone on it before the Final Desicion.
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: Commiters is not the correct spelling it is Committers with 2 T's
Same applies to `Commiter`

Desicion is also not the correct spelling it is Decision simply typo caused by swapping characters

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
...
+
+### How issues are triaged
+
+The Commiters may:
+* close issue that is not reproducible on latest stable release,
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: `close an issue that is not reproducible on latest stable release,`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
...
+* merge issue into another issue,
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: `merge an issue into another issue,`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
...
+* or split issue into multiple ones
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: `or split an issue into multiple issues`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
...
+
+@syuilo reserves Final Desicion rights including whether the project will implement feature and how to implement, but not always exercised.
```
Comment (Mar0xy) [inline CONTRIBUTING.md]: `reserves the Final Decision rights including whether the project will implement the feature and how to implement, but these rights are not always exercised.`

File: CONTRIBUTING.md (hunk 2 above)
Comment (Mar0xy) [reply CONTRIBUTING.md]: `By expressing your intention to work on the Issue`

**Review by Mar0xy — COMMENTED** [review]
Here are just a few suggestions to make it sound more natural in english

**Comment by KisaragiEffective** [comment]
> Here are just a few suggestions to make it sound more natural in english

@Mar0xy thank you for your proof-reading <3

**Comment by syuilo** [comment]
🙏🏿

**Pull request merged**